models/bamf_nnunet_ct_kidney/config/default.yml

```diff
@@ -0,0 +1,37 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet Kidney segmentation (dicom to dicom)
+
+execute:
+- DicomImporter
+- NiftiConverter
+- NNUnetRunner
+- BamfProcessorRunner
+- DsegConverter
+- DataOrganizer
+
+modules:
+  DicomImporter:
+    source_dir: input_data
+    import_dir: sorted_data
+    sort_data: true
+    meta: 
+      mod: '%Modality'
+
+  NNUnetRunner:
+    in_data: nifti:mod=ct
+    nnunet_task: Task779_Kidneys_KIRC
+    nnunet_model: 3d_fullres
+    roi: KIDNEY,NEOPLASM_MALIGNANT_PRIMARY,CYST
```

Comment:
Needs to be `Kidney+NEOPLASM_MALIGNANT_PRIMARY` and `KIDNEY+CYST` to correctly embed these structures into their primary organ.

Reply:
@LennyN95 

Hey Leo, unfortunately this does not work.

`roi: KIDNEY,NEOPLASM_MALIGNANT_PRIMARY,CYST` is the only config that produces an output. Let me know how to proceed with this...